lib/helpers.js

```diff
@@ -60,6 +60,24 @@ exports.checkPropertyName = function(name) {
 };
 
 /**
+ * Validate remote method name. Allows `.` in method name for prototype
+ * @param {String} name The user input
+ * @returns {String|Boolean}
+ */
+exports.checkRemoteMethodName = function(name) {
+  if (name.match(/\./) && !name.match(/^prototype\.(.*)$/)) {
```

Comment:
Is `.` character allowed after `prototype.`? I believe the current code allows that.

I think you need to change the second regexp to `/^prototype\.([^.]*)$/`.